packages/cli/src/commands/deployment/deploy.ts

```diff
@@ -33,6 +33,18 @@ export default class Deploy extends Command {
     queryVersion: Flags.string({description: 'Enter query-version', required: false}),
     dict: Flags.string({description: 'Enter dictionary', required: false}),
     endpoint: Flags.string({description: 'Enter endpoint', required: false}),
+    //indexer set up flags
+    indexer_unsafe: Flags.boolean({description: 'Enable indexer unsafe', required: false}),
```

Comment:
How are these flags used? Would it be `--query_batchSize`?

I think we can change these to camelCase like with queryVersion

Reply:
yes will be like `--query_batchSize`,yup easy to change